admin.ghactivity.php

```diff
@@ -322,3 +332,65 @@ function ghactivity_do_settings() {
 	</div><!-- .wrap -->
 	<?php
 }
+
+/**
+ * GitHub Label Scan section.
+ *
+ * @since 2.1.0
+ */
+function ghactivity_label_scan_callback() {
+	echo '<p>';
+	esc_html_e( 'This button will scan for all the label updates for recorded issues' );
+	echo '</p>';
+
+	echo '<div class="wrap">';
+	echo '<button onclick="triggerLabelScan()">Trigger Label Rescan</button>';
+	echo '</div>';
+
+	?>
+		<script type="text/javascript" >
+		function triggerLabelScan($) {
+			if ( ! confirm( 'This is time-consuming operation. Make sure not to run it more then once!' ) ) {
+				return;
+			}
+			var data = {
+				'action': 'label_scan_action',
+			};
+
+			// since 2.8 ajaxurl is always defined in the admin header and points to admin-ajax.php
+			jQuery.post( ajaxurl, data, function( response ) {
+				alert( 'Got this from the server: ' + response );
+			} );
+		}
+		</script>
+		<?php
+}
+
+add_action( 'wp_ajax_label_scan_action', 'label_scan_action' );
+
+function label_scan_action() {
+	error_log( print_r( 'label_scan_action START!', 1 ) );
+	global $wpdb;
+	$gha = new GHActivity_Calls();
+
+	$post_ids = $wpdb->get_col( "SELECT ID FROM wp_posts WHERE post_type = 'ghactivity_issue' AND post_status = 'publish'" );
```

Comment:
It could be good to use `prepare` here:
https://developer.wordpress.org/reference/classes/wpdb/prepare/